CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2016-09-26 - Release 0.9.15
+
+- Remove invalid dependency from metadata
```

Comment:
please add the other changes as well. You can find a list here: https://github.com/voxpupuli/puppet-yum/compare/v0.9.14...master